modules/end-user-guide/partials/proc_building-custom-intellij-editor-images.adoc

```diff
@@ -0,0 +1,154 @@
+[id="building-custom-intellij-editor-images_{context}"]
+= Building custom IntelliJ editor images
+
+This procedure describes how to build an IntelliJ-based editor image using a custom archive and a custom image name.
+
+.Prerequisites
+
+* The `git`, `jq`, and `wget` tools are available.
+
+* Docker version 18.09 or greater, supporting link:https://docs.docker.com/develop/develop-images/build_enhancements/[Docker BuildKit].
+
+* Java Development Kit (JDK) version 11 is available.
+
+* On the macOS platform: the `+gnu-getopt+` tool is available.
+
+* The build platform has at least 2 GB of RAM.
+
+.Procedure
+
+. Get a local copy of the link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains Projector Editor Images repository].
++
+----
+$ git clone https://github.com/che-incubator/jetbrains-editor-images
+$ cd jetbrains-editor-images
+----
+
+. Build the container using build parameters:
++
+[subs="+quotes,macros,attributes"]
+----
+$ ./projector build --tag __<tag>__ --url __<URL>__
+----
++
+`--tag __<tag>__`::
+Tag to apply to the image after build.
```

Comment:
The tag to apply to the image after build.

Reply:
`tag` is a noun and a verb. I guess `The` makes it clearer that it is meant as a noun here.

Reply:
Done.